src/current/v23.1/operational-faqs.md

```diff
@@ -81,9 +81,13 @@ When MVCC garbage is deleted by garbage collection, the data is still not yet ph
 
 {% include {{page.version.version}}/storage/free-up-disk-space.md %}
 
-## How can I free up disk space quickly?
+## How can I free up disk space when dropping a table?
```

Comment:
```suggestion
## How can I free up disk space that was used by a dropped table?
```

Reply:
I don't think i can take this one because as we discussed in [this previous comment by Adam](https://github.com/cockroachdb/docs/pull/19087#discussion_r1827011703) we're trying to make it super duper clear that the user needs to prepare to free up the disk space _when_ (really, before) dropping the table because they have to check the range IDs before issuing the `DROP TABLE`

 whereas "that was used by a dropped table" could imply that this is how you GC the ranges of something you dropped in the past which this is not that
 
 but which we're gonna have to document separately via https://cockroachlabs.atlassian.net/browse/DOC-11930 until we get https://github.com/cockroachdb/cockroach/issues/105808